pytorch_lightning/strategies/tpu_spawn.py

```diff
@@ -346,3 +348,9 @@ def register_strategies(cls, strategy_registry: Dict) -> None:
         strategy_registry.register(
             "tpu_spawn_debug", cls, description="TPUSpawn Strategy with `debug` as True", debug=True
         )
+
+        strategy_registry.register(
```

Comment:
@four4fish Could this be moved to the base class?

Reply:
Not really, I think it won't work. I have added into https://github.com/PyTorchLightning/pytorch-lightning/issues/11449 follow up item 3. If we have it in base class, the cls will be the base class instead of the class inheritance the base class.